index.html

```diff
@@ -39,6 +42,8 @@
         document.getElementById('sel-themes').addEventListener('change', function () {
             cssLink.href = 'bower_components/medium-editor/dist/css/' + this.value + '.css';
         });
+
+        document.getElementById('version').innerHTML = 'This demo uses medium-editor v'+MediumEditor.version.toString();
```

Comment:
the casting makes the implicit toString happen. not a big deal really. just wanted to see how you'd used it.